generateBins.py

```diff
@@ -93,6 +54,22 @@ def process(configuration, gisPath = ""):
         print("Treatments need binning, not currently supported")
         exit(1)
 
+        # getting data ready for binning
+        # data should be 1-dimensional array, python list or iterable
+        treat = np.loadtxt(TREATMENT_FILE, skiprows=6)
+        treat_1d = treat.flatten()
+
+        # GVF Implementation
+        gvf = 0.0
+        nclasses = 20
+        while gvf < 0.8:
+            gvf_t = goodness_of_variance_fit(treat_1d, nclasses)
+            nclasses += 1
+        print("The value of Goodness of Variance fit is:", gvf)
+        if (gvf < 0.7):
+            print("Warning: GVF too low")
```

Comment:
Don't worry about the treatments for now. It's not needed for RWA and for BFA we used a different approach.